openedx/core/djangoapps/content/course_overviews/migrations/0002_add_days_early_for_beta.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'CourseOverview.days_early_for_beta'
+        db.clear_table('course_overviews_courseoverview')
```

Comment:
Please add a bold/obvious comment of why we are clearing the table.  Otherwise, readers may not notice that the auto-generated migration file had to be manually edited.